src/components/TopBar/TopBar.tsx

```diff
@@ -418,6 +438,51 @@ export class TopBar extends React.Component<{
               gap: '4px',
             }}
           >
+            {this.props.showInviteButton && (
+              <Popup
+                onClose={() => this.setState({ inviteLinkCopied: false })}
+                trigger={
+                  <Button
+                    color="green"
+                    icon
+                    labelPosition="left"
+                    fluid
+                    className="toolButton"
+                    style={{ minWidth: '12em' }}
+                  >
+                    <Icon name="add user" />
+                    Invite Friends
+                  </Button>
+                }
+                content={
+                  <Container fluid textAlign="left">
```

Comment:
i think we should use the modal component for this? and the state could move into the modal component